src/NumberTheory/GaloisGrp/Group.jl

```diff
@@ -7,7 +7,7 @@ minimal supergroups, ie. it fails in ``C_4``.
 function maximal_subgroup_chain(G::PermGroup, U::PermGroup)
   l = [G]
   while order(l[end]) > order(U)
-    m = maximal_subgroups(l[end])
+    m = reduce(vcat, map(collect, maximal_subgroup_classes(l[end])))
```

Comment:
Should this use `maximal_subgroups` again? Possibly combined with `collect`:
```suggestion
    m = collect(maximal_subgroups(l[end]))
```